Templates/AppSource App/.github/workflows/CICD.yaml

```diff
@@ -66,7 +65,25 @@ jobs:
         uses: microsoft/AL-Go-Actions/ReadSettings@main
         with:
           shell: powershell
-          get: type, powerPlatformSolutionFolder
+          get: type,powerPlatformSolutionFolder,useSubModules
+
+      - name: Read subModules token
+        id: ReadSubModulesToken
+        if: env.useSubModules != 'false' && env.useSubModules != ''
+        uses: microsoft/AL-Go-Actions/ReadSecrets@main
+        with:
+          shell: ${{ inputs.shell }}
+          gitHubSecrets: ${{ toJson(secrets) }}
+          getSecrets: '-subModulesToken'
```

Comment:
What's the dash in the beginning for?

Reply:
For asking readsecrets to not base64 encode the secret